lib/vtt.js

```diff
@@ -1204,8 +1204,17 @@
     })();
   };
 
-  WebVTT.Parser = function(window, decoder) {
+  WebVTT.Parser = function(window, vttjs, decoder) {
```

Comment:
I'd rather pass an options object here instead of doing the dance below. It'll break the API, but we're not at 1.0 yet so I'd be comfortable with that.